static/css/components/header-bar.less

```diff
@@ -699,3 +854,34 @@ div.search-facet:focus-within {
     }
   }
 }
+
+@media only screen and (max-width: @width-breakpoint-tablet) {
+  // stylelint-disable-next-line selector-max-specificity
+  #header-bar {
+    position: sticky;
+    top: 0;
+    background: linear-gradient(to bottom, @gradient-start 50px, @gradient-end 218px) !important;
```

Comment:
Ah, this should be `linear-gradient(to bottom, #f6f1e6, #f1e9d7)`. I just realized why it wasn't looking like a gradient; it starts at `50px`, which is larger than the header bar! So it's all getting set to the first colour.

Also tweaked the colour values a bit. Looks like I had modified them a little from what was on the explore page.

Reply:
Also note important isn't needed